dev/data/fineweb.py

```diff
@@ -80,6 +96,15 @@ def tokenize(doc):
     all_tokens_np = np.empty((args.shard_size,), dtype=np.uint16)
     token_count = 0
     progress_bar = None
+
```

Comment:
doesn't `def tokenize` break because:

```python
all_tokens_np = np.empty((args.shard_size,), dtype=np.uint16)
```

i.e. the init is using uint16